src/main/resources/BridgeServer2.conf

```diff
@@ -146,6 +146,9 @@ dev.udd.sqs.queue.url = https://sqs.us-east-1.amazonaws.com/649232250620/Bridge-
 uat.udd.sqs.queue.url = https://sqs.us-east-1.amazonaws.com/649232250620/Bridge-WorkerPlatform-Request-uat
 prod.udd.sqs.queue.url = https://sqs.us-east-1.amazonaws.com/649232250620/Bridge-WorkerPlatform-Request-prod
 
+#Bridge Participant Roster Download Worker SQS queues
+workerPlatform.request.sqs.queue.url = https://sqs.us-east-1.amazonaws.com/420786776710/Bridge-WorkerPlatform-Request-local
```

Comment:
Should also specify for dev, uat, and prod. Check the AWS console to get the right URL.

Reply:
Maybe I don't have access to the uat and prod queues - the only WorkerPlatform queues I see end in alxdark, develop, dwanejeng, local, and staging. There are also some dead letter queues.

Reply:
However it seems that the urls differ by just the "-develop" or "-local" at the end. I'll put in some place holders and could you verify if they match up with the queues you see on your end?